src/ensembl/src/shared/components/select/Select.tsx

```diff
@@ -115,18 +114,13 @@ const Select = (props: SelectProps) => {
   };
 
   const handleKeyPress = (event: KeyboardEvent) => {
-    if (
-      !(
-        focusRef.current &&
-        [keyCodes.ENTER, keyCodes.ESC].includes(event.keyCode)
-      )
-    ) {
+    if (!focusRef.current) {
       return;
     }
 
-    if (event.keyCode === keyCodes.ENTER) {
+    if (event.key === 'Enter') {
       openPanel();
-    } else if (event.keyCode === keyCodes.ESC) {
+    } else if (event.key === 'Escape') {
```

Comment:
Do we need the `else` here?

Reply:
Don't think so; we'll just ignore those key presses.